### PR TITLE
Fix rl check

### PR DESCRIPTION
**Pull request by rickducott**
# Description

Please include a summary of the changes.

This bug fixes a broken rate limit check in glooctl. 
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/3030

### CODE REVIEW

File: changelog/v1.4.0-beta12/fix-check.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+changelog:
+  - type: FIX
+    issueLink: https://github.com/solo-io/gloo/issues/3030
+    description: Fixes a bug where the help comment in the stats would be picked up as an actual stat, causing the rate limit connected state check to fail.
```
Comment (kdorosh) [inline changelog/v1.4.0-beta12/fix-check.yaml:4]: can we make this more clear to users, that this affects `glooctl check`

**Comment by rickducott** [comment]
/kick 
ingress flake / discovery pod not up: https://storage.googleapis.com/solo-public-build-logs/logs.html?buildid=980df814-d224-44a5-b55f-9582a33a147c

**Comment by rickducott** [comment]
/kick

ingress test timing out at 120s: https://storage.googleapis.com/solo-public-build-logs/logs.html?buildid=2cf90c7c-d3e0-4517-9773-24c876c2ac7c

**Pull request merged**